src/StudyViewer/StudyViewer.jsx

```diff
@@ -127,9 +127,10 @@ class StudyViewer extends React.Component {
           });
         }
         if (desc) {
-          return (bVal > aVal) ? 1 : -1;
+          return bVal.localeCompare(aVal);
         }
-        return (aVal > bVal) ? 1 : -1; // asc by default
+        // asc by default
+        return aVal.localeCompare(bVal);
```

Comment:
since we can configure any field to sort on, i think we need it to work for types other than str, mostly int - can we add a type check